### ISSUE TITLE
Jitter (non-smooth) animations. Most noticable at slower speeds.

### ISSUE THREAD

**Issue by sambernaerdt** [mwydmuch/ZoomVideoComposer#16]
First of all: Thanks again for putting this together.

I did notice that at slower speeds, the animation is not super smooth. It looks like it gets rounded to some values. As soon as the speed increases, it gets less noticable.

Some examples where it is visible:
- https://www.youtube.com/watch?v=8MAlM75bYGs&t=2s, created with easeInOutSine
- https://www.youtube.com/watch?v=CLjxMRtUJRQ, created with linearEaseInOut

**Comment by mwydmuch** [comment]
Yes, I'm aware of that, it happens because image cropping in both image backends requires rounding to the integer pixel coordinates. Let me think about how to solve it in the best possible way.
I guess the one resource-costly solution is to increase the internal resolution of the images, this will give the best result, and smooth animation. The second option is to better control rounding, sacrificing a bit of smoothness (no jitter but possibly visible pixel size shifts).

**Comment by sambernaerdt** [comment]
IMHO: I wouldn't mind the sacrifice of longer render times if that is a quick fix to increase the smoothness. I have no clue at what magnitudes of render time increase we are looking here, but I guess it could work when it's an argument to opt-in when desired.

- Set it at 1 (or whatever the default is) to do a test run until everything looks good.
- Set it to a higher value and be patient for the final render.
